app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ReservationsController < ApplicationController
+  def create
+    reservation = Reservation.new(reservation_params)
+    reservation.confirmed!
```

Comment:
are we automatically confirming the reservation? or is this just temporary until we have the function for the host to confirm